### PR TITLE
Update iOS push extras docs

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

**Comment by Bucimis** [comment]
@EmilyNecciai please merge :)

File: _docs/_developer_guide/platform_integration_guides/ios/push_notifications/customization.md [hunk 1]
```diff
@@ -151,17 +151,7 @@ func application(_ application: UIApplication,
...
-```objc
-- (void)handleExtrasFromPush:(NSDictionary *)notification {
-  NSLog(@"A push was received");
-  if (notification != nil && [notification objectForKey:@"Some_Key"] !=[NSNull null] && [[notification objectForKey:@"Some_Key"] length] > 0 ) {
-    NSString *value = [NSString stringWithFormat:@"%@",[notification objectForKey:@"Some_Key"]];
-    // Here based on the extras key-value pair, you can run some arbitrary code.
-  }
-}
-```
+Braze allows you to send custom-defined string key-value pairs, known as extras, along with a push notification to your application. Extras can be defined on the dashboard or from the API and will be available as string typed key-value pairs within the notification dictionary passed to your push delegate implementations.
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/push_notifications/customization.md]: "can be defined on the dashboard or `via*` API?

Comment (hokstuff) [reply _docs/_developer_guide/platform_integration_guides/ios/push_notifications/customization.md]: Also, `will be available as string typed key-value pairs within the notification dictionary` sounds a bit vague to me. This wording makes it unclear if both the key and the value are strings, or if it's just the key

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/push_notifications/customization.md]: 1) done
2) removed "string typed" - they're alluded to being strings earlier in the sentence and elsewhere in docs

**Pull request merged**